### PR TITLE
l10n: Extract a space from the translation

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by tcitworld — CHANGES_REQUESTED** [review]
Nope, the variable should be brought into the translation instead of concatening.

**Comment by Valdnet** [comment]
Corrected variable, but I don't know if the bot will scan the text string correctly.

@tcitworld Is it ok now?

File: lib/Controller/EmailController.php [hunk 1]
```diff
@@ -134,7 +134,7 @@ public function sendEmailPublicLink(string $recipient,
 		}
 
 		return new JSONResponse([
-			'message' => $this->l10n->t('Successfully sent email to ' . $recipient),
+			'message' => $this->l10n->t('Successfully sent email to' . ' ' . $recipient),
```
Comment (raimund-schluessler) [inline lib/Controller/EmailController.php:137]: @Valdnet I think @tcitworld meant something like this (not tested):
```suggestion
			'message' => $this->l10n->t('Successfully sent email to %1$s', $recipient),
```

**Review by rakekniven — APPROVED** [review]

File: lib/Controller/EmailController.php [hunk 2]
```diff
@@ -134,7 +134,7 @@ public function sendEmailPublicLink(string $recipient,
 		}
 
 		return new JSONResponse([
-			'message' => $this->l10n->t('Successfully sent email to ' . $recipient),
+			'message' => $this->l10n->t('Successfully sent email to %1$s', $recipient),
```
Comment (tcitworld) [inline lib/Controller/EmailController.php:137]: CI says it should be an array.
```suggestion
			'message' => $this->l10n->t('Successfully sent email to %1$s', [$recipient]),
```

**Review by tcitworld — CHANGES_REQUESTED** [review]

**Review by tcitworld — APPROVED** [review]

**Pull request merged**